### PR TITLE
# Web App - hosts

### PR DESCRIPTION
**Pull request by szymonqs**
Create list of hosts

### CODE REVIEW

File: PZ/WebClient/models.py [hunk 1]
```diff
@@ -6,3 +6,9 @@ class Monitors(models.Model):
     monitor_name = models.CharField(max_length=200)
     monitor_domain = models.URLField()
     user_id = models.ForeignKey(User)
+
+class Hosts(models.Model):
+    monitor_id_id = models.IntegerField()
```
Comment (Dorrin93) [inline PZ/WebClient/models.py]: _id_id?

File: PZ/WebClient/models.py [hunk 2]
```diff
@@ -6,3 +6,9 @@ class Monitors(models.Model):
     monitor_name = models.CharField(max_length=200)
     monitor_domain = models.URLField()
     user_id = models.ForeignKey(User)
+
+class Hosts(models.Model):
+    monitor_id_id = models.IntegerField()
+    host_name = models.CharField(max_length=200)
```
Comment (Dorrin93) [inline PZ/WebClient/models.py]: Names are local for classes, so I don't think `host_` before variable names are necessary, what do you think @PatrykMatwijczuk ?

**Review by Dorrin93 — APPROVED** [review]
Besides naming, looks fine to me

File: PZ/WebClient/models.py (hunk 2 above)
Comment (PatrykMatwijczuk) [reply PZ/WebClient/models.py]: Yep, I fully agree with @Dorrin93

File: PZ/WebClient/models.py [hunk 3]
```diff
@@ -6,3 +6,9 @@ class Monitors(models.Model):
     monitor_name = models.CharField(max_length=200)
     monitor_domain = models.URLField()
     user_id = models.ForeignKey(User)
+
+class Hosts(models.Model):
+    monitor_id_id = models.IntegerField()
+    host_name = models.CharField(max_length=200)
+    host_description = models.CharField(max_length=200)
+    host_measurements = models.CharField(max_length=200)
```
Comment (PatrykMatwijczuk) [inline PZ/WebClient/models.py]: measurements is not CharFiled

**Review by PatrykMatwijczuk — APPROVED** [review]

**Pull request merged**